frontend/src/metabase/collections/components/PinnedQuestionCard/PinnedQuestionLoader.tsx

```diff
@@ -45,7 +45,7 @@ const PinnedQuestionLoader = ({
   return (
     <Questions.Loader id={id} loadingAndErrorWrapper={false}>
       {({ loading, question: card }: QuestionLoaderProps) => {
-        if (loading) {
+        if (loading || !card.dataset_query) {
```

Comment:
@ranquild 
Out of curiosity, when/how does card exist without the `dataset_query`? In theory, we cannot create a card without that query, right?

Reply:
@nemanjaglumac it's the result of loading pinned collection items into the redux state https://github.com/metabase/metabase/blob/426da4a3dc5afc27ff7607423aae4a54b03097c2/frontend/src/metabase/collections/containers/CollectionContent.jsx#L181. The loader puts data into the same state path as regular card objects

Reply:
Thanks